portal_share_custom_filter/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_ir_filters_group_auction_user,ir.filters.group,base_custom_filter.model_ir_filters_group,base.group_portal,1,0,0,0
```

Comment:
I think we should use auction seller group instead of portal group.

Reply:
I did this so as not to add dependence on custom modules.

But giving the controller part to the action user may also be fine, so check it out.